paddle/cinn/hlir/dialect/operator/ir/attribute_storage.h

```diff
@@ -77,5 +77,27 @@ struct GroupInfoAttributeStorage : public pir::AttributeStorage {
   ParamKey data_;
 };
 
+struct CUDAJITInfoAttributeStorage : public pir::AttributeStorage {
```

Comment:
```suggestion
struct JITInfoAttributeStorage : public pir::AttributeStorage {
```
这里是不是可以移除 CUDA 前缀，后续还有其他硬件，本质上拿到的都是函数指针